src/VCreditCard.vue

```diff
@@ -22,43 +22,64 @@
             <div class="vcc-col">
                 <div class="vcc-form-group">
                     <label class="vcc-label" for="holder">{{$t('form.holder')}}</label>
-                    <input type="text" class="vcc-control" id="holder" v-model="fullName" autocomplete="cc-name"/>
+                    <input type="text" class="vcc-control"
+                           :data-error="(cardErrors.cardFullName)"
+                           id="holder" v-model="fullName" autocomplete="cc-name"/>
+                    <div v-if="cardErrors.cardFullName" class="error">
```

Comment:
You should use `vcc-` prefix for class attributes. Suggested class name is `vcc-input-error`